packages/inter-protocol/src/price/roundsManager.js

```diff
@@ -364,10 +364,9 @@ export const makeRoundsManagerKit = defineDurableFarClassKit(
         }
 
         /** @type {bigint | undefined} */
+        // @ts-expect-error faulty inference
         const newAnswer = calculateMedian(
-          details
-            .get(roundId)
-            .submissions.filter(sample => isNat(sample) && sample > 0n),
+          details.get(roundId).submissions.filter(sample => isNat(sample)),
```

Comment:
this code change slipped in, though it shouldn't affect behavior because `isNat` is only true if the argument is >0.

Reply:
`isNat` requires `>= 0n` but not `> 0n`:

```
> isNat(0n)
true
> 
```

Reply:
thanks for catching!